crates/oxc_transformer/src/lib.rs

```diff
@@ -90,11 +92,12 @@ impl<'a> Transformer<'a> {
         scopes: ScopeTree,
         program: &mut Program<'a>,
     ) -> TransformerReturn {
-        let allocator = self.ctx.ast.allocator;
+        let allocator = self.allocator;
+        let ast_builder = AstBuilder::new(allocator);
 
         let mut transformer = TransformerImpl {
             x0_typescript: TypeScript::new(self.options.typescript, &self.ctx),
-            x1_react: React::new(self.options.react, &self.ctx),
+            x1_react: React::new(self.options.react, ast_builder, &self.ctx),
```

Comment:
@shulaoda This is how I solved the problem. This is pretty hacky, but it works.

We are intending to revamp the interface to `Transformer` soon, so hopefully we can come up with a more ergonomic solution then.

Reply:
👍